robottelo/cli/hammer.py

```diff
@@ -61,6 +64,13 @@ def _normalize_obj(obj):
 
 def parse_csv(output):
     """Parse CSV output from Hammer CLI and convert it to python dictionary."""
+    try:
+        warning_index = output.index(
+            'Puppet and OSTree will no longer be supported in Katello 3.16'
```

Comment:
3.16 is needed compulsory here? maybe tomorrow it will change to 3.17 and in that case, you will change the text every time. Not sure let me if I'missing anything. 

My intention is here that 3.16 if that is coming from the config file in DEV code then it could change to 3.17. if it is hard coded then it is fine in Dev Code.

Reply:
String is hardcoded in [repositories_controller.rb](https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/repositories_controller.rb#L223).

I don't think it makes any difference if string is hardcoded or comes from configuration. If devs now decide to postpone deprecation, they can change string easily. Since 3.16 refers to future katello version, I wouldn't worry about it changing dynamically and often. If my assumption turns out to be wrong, we can think about better approach (there will be increased number of failures in automation, we will notice that).